zerver/tests/test_signup.py

```diff
@@ -274,7 +274,7 @@ def test_register(self):
         with queries_captured() as queries:
             self.register(self.nonreg_email('test'), "test")
         # Ensure the number of queries we make is not O(streams)
-        self.assert_length(queries, 66)
+        self.assert_length(queries, 106)
```

Comment:
This is a really big increase and will cause performance problems.  I think we want to do a `bulk_create` when doing a bulk subscribe to avoid increasing this number by dozens.